library/init/data/option_t.lean

```diff
@@ -36,16 +36,15 @@ instance {m : Type u → Type v} [monad m] : monad (option_t m) :=
  pure_bind := begin
    intros,
    dsimp [option_t_bind, option_t_return],
-   dsimp [return, pure, bind], -- TODO: fix signature of `pure_bind`
    rw [monad.pure_bind], apply rfl
  end,
  bind_assoc := begin
    intros,
-   dsimp [option_t_bind, option_t_return, bind],
+   dsimp [option_t_bind, option_t_return],
    rw [monad.bind_assoc],
    apply congr_arg, apply funext, intro x,
    cases x,
-   { dsimp [option_t_bind._match_1, return, pure],
+   { dsimp [option_t_bind],
```

Comment:
There are a lot of simp calls where this PR removes the `._match_1`.  Is this a change that became possible through the structure refactoring, or is this just general cleanup?